### PR TITLE
imporving template view

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
@itkg-nanne

File: BackofficeBundle/Resources/public/coffee/page/makeSortable.coffee [hunk 1]
```diff
@@ -10,52 +6,17 @@ refreshUl = (ul) ->
...
-    childs.each ->
-      $(this).css direction, (100 / nbrChildren) + "%"
-      $(this).css (if direction is "height" then "width" else "height"), "auto"
-      return
+    return
   return
 
-makeSortable = (el, duplicate) ->
+
+makeSortable = (el) ->
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/page/makeSortable.coffee]: You change the function signature, that's ok for me, but I don't see where you're changing the function calling

**Comment by itkg-rboessel** [comment]
@nicolasThal @itkg-ngilain @itkg-nanne

**Comment by itkg-ngilain** [comment]
I have no idea of the implications of these changes, but I see nothing bad in particular, so :+1: :)

**Comment by nicolasThal** [comment]
I will checkout the branch on my local installation to see the modification first :)

**Comment by nicolasThal** [comment]
works fine on my local computer : :+1:

**Pull request merged**